README.md

```diff
@@ -625,46 +657,47 @@ Body:
             "securityGroup": "<uuid>",
             "azureDiskType": "enum",
             "vmInstance": "string"
-        }
+        },
+        "nicOverrides": [
+          {
+            "name": "<string>",
+            "vmIdentifier": "string",
+            "failover": {
+              "ipConfig": {
+                "staticIp": "<string>",
+                "isDhcp": "<bool>",
+                "subnetMask": "<string>",
+                "gateway": "<string>",
+                "primaryDns": "<string>",
+                "secondaryDns": "<string>",
+                "dnsSuffix": "<string>"
+              },
+              "networkId": "<uuid>",
+              "shouldReplaceMacAddress": "<bool>"
+            },
+            "test": {
+              "ipConfig": {
+                "staticIp": "<string>",
+                "isDhcp": "<bool>",
+                "subnetMask": "<string>",
+                "gateway": "<string>",
+                "primaryDns": "<string>",
+                "secondaryDns": "<string>",
+                "dnsSuffix": "<string>"
+              },
+              "networkId": "<uuid>",
+              "shouldReplaceMacAddress": "<bool>"
+            }
+          }
+        ]
+    }
+   "protectedEntities": {
+                "items": [
+                   {
```

Comment:
here there is no type?

Reply:
As to what we checked, the type is only in GET